xml/security-troubleshooting_tls.xml

```diff
@@ -102,6 +102,14 @@ ansible-playbook -i hosts/verb_hosts monasca-start.yml</screen>
    Percona clusters.
   </para>
   <procedure>
+   <step>
+    <para>
+      Determine if the TLS certificates for MySQL / Percsona have expired.
+    </para>
+    <screen>&prompt.ardana;cd /etc/mysql/
+&prompt.ardana;/etc/mysql # openssl x509 -noout -text -in control-plane-1-mysql-internal-cert.pem | grep After
```

Comment:
To be precise, you may want to do `openssl x509 -noout -enddate -in control-plane-1-mysql-internal-cert.pem` instead.

Reply:
updated, please take another look, thanks!